e2e/cypress/integration/enterprise/saml/saml_automated_spec.js

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. #. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @enterprise @saml
+// Skip:  @headless @electron @firefox // run on Chrome (headed) only
+
+import users from '../../../fixtures/saml_users.json';
+
+//Manual Setup required: Follow the instructions mentioned in the mattermost/platform-private/config/saml-okta-setup.txt file
+context('LDAP SAML - Automated Tests (SAML TESTS)', () => {
+    const loginButtonText = 'SAML';
+
+    const regular1 = users.regulars['samluser-1'];
+
+    const {
+        oktaBaseUrl,
+        oktaMMAppName,
+        oktaMMEntityId,
+    } = Cypress.env();
+    const idpUrl = `${oktaBaseUrl}/app/${oktaMMAppName}/${oktaMMEntityId}/sso/saml`;
+    const idpMetadataUrl = `${oktaBaseUrl}/app/${oktaMMEntityId}/sso/saml/metadata`;
+
+    const newConfig = {
+        SamlSettings: {
+            Enable: true,
+            EnableSyncWithLdap: false,
+            EnableSyncWithLdapIncludeAuth: false,
+            Verify: true,
```

Comment:
Currently all the test passes if Verify flag is set to false. Can you please confirm if this is a bug or if it happens on localhost because the server is not using encryption

Reply:
@hahmadia please ignore this. I just reinstalled the xmlsec library again on my localhost and it started to work fine again. Even @saturninoabril had the same issue and it worked fine once it was installed on his system.